README.md

```diff
@@ -20,15 +22,57 @@ PGTABLENAME=space2stats
 ```
 
 - Load our dataset into the database
+
 ```
 ./postgres/download_parquet.sh
-python postgres/chunk_parquet.py
-./postgres/load_parquet_chunks.sh
+./load_to_prod.sh
 ```
 
 > You can get started with a subset of data for NYC with `./load_nyc_sample.sh` which requires changing your `db.env` value for `PGTABLENAME` to `space2stats_nyc_sample`.
 
 - Access your data using the Space2statS API! See the [example notebook](notebooks/space2stats_api_demo.ipynb).
 
+## Usage as a module
+
+The module can be installed via `pip` directly from Github:
+
+```
+pip install "git+https://github.com/worldbank/DECAT_Space2Stats.git#subdirectory=space2stats_api/src"
```

Comment:
added a missing ". Also, needed to install via specific branch to get the changes (obviously): `pip install "git+https://github.com/worldbank/DECAT_Space2Stats.git@refactor/more-module-friendly#subdirectory=space2stats_api/src"`. The import statement below worked well!